private/src/scripts/editor/blocks/header/DisplayComponent.jsx

```diff
@@ -300,6 +313,20 @@ class DisplayComponent extends Component {
               )}
             </div>
           )}
+          {this.state.videoUrl && attributes.type === 'video' && (
+            <div className="image-metadata">
+              {shouldShowImageCaption && (
+                <span className="image-metadataItem image-caption">
+                  {this.state.videoCaption}
+                </span>
+              )}
+              {shouldShowImageCredit && (
+                <span className="image-metadataItem image-copyright">
+                  {this.state.videoDescription}
+                </span>
+              )}
+            </div>
+          )}
```

Comment:
this + the image meta output should perhaps be pulled out into a separate component

Reply:
resolved